src/sentry/replays/usecases/ingest/dead_click.py

```diff
@@ -31,7 +31,7 @@ def report_dead_click_issue(project_id: int, replay_id: str, event: SentryEvent)
         extra_event_data={
             "contexts": {"replay": {"replay_id": replay_id}},
             "level": "warning",
-            "tags": {"replayId": replay_id},
+            "tags": {"replayId": replay_id, "url": payload["data"]["url"]},
```

Comment:
Will this be the FQDN or just path?

Reply:
url always guaranteed to exist i assume?

Reply:
@billyvg Can confirm but in my observation of the logs yes it seems guaranteed to exist for timeouts endReasons.